core/app/[locale]/(default)/webpages/[id]/contact/page.tsx

```diff
@@ -65,18 +65,20 @@ const getWebPage = cache(async (id: string): Promise<ContactPage> => {
   };
 });
 
-async function getWebPageBreadcrumbs(id: string): Promise<Breadcrumb[]> {
+async function getWebPageBreadcrumbs(id: string): Promise<BreadcrumbWithId[]> {
   const webpage = await getWebPage(id);
   const [, ...rest] = webpage.breadcrumbs.reverse();
   const breadcrumbs = [
     {
       label: 'Home',
```

Comment:
🍹 Same here along with some other files not commented on

Reply:
Created a separate ticket for this.